tests/test/scheduler/test_scheduler.cpp

```diff
@@ -418,16 +417,13 @@ TEST_CASE_METHOD(SlowExecutorFixture,
     SECTION("Functions") { execMode = faabric::BatchExecuteRequest::FUNCTIONS; }
 
     // Set up snapshot if necessary
-    faabric::util::SnapshotData snapshot;
     faabric::snapshot::SnapshotRegistry& snapRegistry =
       faabric::snapshot::getSnapshotRegistry();
 
-    std::unique_ptr<uint8_t[]> snapshotDataAllocation;
+    size_t snapSize = 1234;
```

Comment:
I know it was like this before, but shouldn't snapshot sizes be multiples of the OS page size (4096)?

Reply:
I don't think so. AFAICT there's nothing in the implementation that restricts them to being page-aligned, although the use of `mmap` under the hood will cause the underlying memory and mappings to be rounded up to the nearest OS page. 

When used in Faasm they will all be multiples of the wasm page size (64k), so in practice they will all be OS-page aligned, but I don't think we necessarily need to bake that into the API at the moment.